aspnetcore/blazor/fundamentals/static-files.md

```diff
@@ -193,6 +193,44 @@ For examples of how to address the policy violation with Subresource Integrity (
 
 :::moniker-end
 
+:::moniker range=">= aspnetcore-10.0"
+
+### Fingerprint static assets in standalone Blazor WebAssembly apps
+
+In standalone Blazor WebAssembly apps during build/publish, the framework overrides placeholders in `index.html` with values computed during build to fingerprint static assets. A fingerprint is placed into the `blazor.webassembly.js` script file name, and an import map is generated for other .NET assets.
```

Comment:
Regarding the importmap, I can contain other assets as well, if they are javascript modules. Do have it covered for hosted mode using `<ImportMap />` component?

Related https://github.com/dotnet/runtime/issues/74815#issuecomment-2737171054.

Reply:
It works differently for hosted and standalone mode. In hosted mode all JS modules are soft fingerprinted, meaning that the file on disk don't have fingerprint and only aspnetcore server knows about URLs with fingerprint. We hard fingerprint .NET assets, meaning file on disk has a fingerprint. In standalone mode, there is no aspnetcore server and so only hard fingerprinted assets have fingerprint. And so if users want to fingerprint their JS modules and put them to importmap, they need to opt-in (as described in the linked comment). That is something we don't have convered with docs, am I right?

Reply:
Not covered. I just saw the linked remark.

What is meant by "hosted" or "hosted mode?" Do you mean the `.Client` project of a BWA, or does that mean the old hosted Blazor WebAssembly solution/template ... or both?

Reply:
Both